src/test/ui/lint/issue-47390-unused-variable-in-struct-pattern.rs

```diff
@@ -35,6 +35,10 @@ fn main() {
         endless_and_singing: true
     };
 
+    let mut mut_unused_var = 1;
```

Comment:
Not sure if that works, but you can try adding a `// run-rustfix` comment to the top of the file. That'll show us the file after applying the suggestions